src/hooks/screen/useCreateChat.ts

```diff
@@ -124,6 +124,22 @@ const useCreateChat = () => {
     }
   };
 
+  const createAnonymousChat = async (selectedUser) => {
+    try {
+      setLoadingCreateChat(true);
+      const response = await getOrCreateAnonymousChannel(selectedUser?.user?.userId);
+      const chatData = createChannelJson(response, selectedUser);
+      const channelList = ChannelList.fromMessageAnonymouslyAPI(chatData);
+      await channelList.saveIfLatest(localDb);
+      handleMemberSchema(response);
+      goToChatScreen(channelList, GROUP_INFO);
+    } catch (e) {
+      console.log({e}, 'error create chat');
+    } finally {
+      setLoadingCreateChat(false);
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `createAnonymousChat` function is missing error handling. Currently, if an error occurs, it's logged to the console but not handled in any other way. This could lead to a poor user experience if something goes wrong during chat creation. Consider adding error handling that informs the user when an error occurs.
```diff
     } catch (e) {
-      console.log({e}, 'error create chat');
+      SimpleToast.show('Failed to create anonymous chat');
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->